src/raftstore/store/peer.rs

```diff
@@ -719,9 +720,9 @@ impl Peer {
         if duration >= self.cfg.max_leader_missing_duration.0 {
             // Resets the `leader_missing_time` to avoid sending the same tasks to
             // PD worker continuously during the leader missing timeout.
-            self.leader_missing_time = None;
+            self.leader_missing_time = Some(Instant::now());
```

Comment:
seem the comment is stale, so here the same tasks can be sent continuously?

Reply:
No, it will resend the requests only after another max missing time.

Reply:
How? The comment says
>  avoid sending the same tasks to PD worker continuously ***during*** the leader missing timeout.